src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -24,194 +24,28 @@ Monitor your Flutter application with our New Relic Flutter agent. Capture Dart
-
-</Step>
-
-<Step>
-### (Android-only) Configure your Android app [#configure-android]
-
-If you have an Android-native app, you'll need make the following changes in your Android app:
 
-1. Add the following changes to your app's `android/build.gradle` file:
-
-```dart
-buildscript {
-        ...
-        repositories {
-            ...
-            mavenCentral()
-        }
-        dependencies {
-            ...
-            classpath "com.newrelic.agent.android:agent-gradle-plugin:${latest_android_version}"
-        }
-    }
-```
-
-2. Apply the `newrelic` plugin to the top of your `android/app/build.gradle` file:
-
-```dart
-apply plugin: "com.android.application"
-apply plugin: 'newrelic' // <-- add this
-```
-
-3. Add `INTERNET` and `ACCESS_NETWORK_STATE` permissions in your `AndroidManifest.xml` file:
-
-```dart
-<uses-permission android:name="android.permission.INTERNET" />
-<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-```
-
-And you're done! You should start seeing data in New Relic within 5 minutes.
-</Step>
-
-</Steps>
-
-## View your Flutter app data in New Relic [#view-data]
-
-Now that you've got the Flutter agent sending data to New Relic, it's time to view your app data in New Relic.
-
-Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile >** (select your Flutter app) **> Summary** to see a summary of your data over time. 
-
-With the Flutter agent, you have access to all mobile monitoring features, such as tracking crashes, HTTP requests, and handled exceptions. Check out these feature pages for more info:
-* [Mobile app summary](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-summary-page/#details)
-* [Handled exceptions](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes)
-* [Crash event trails](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/mobile-crash-event-trail)
-* [HTTP requests](/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-requests-page)
-* [Set up alerts](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-monitoring-alert-information)
-
-<Callout variant="tip" title="Troubleshoot missing data in the UI">
-
-Having trouble finding HTTP data in the UI?
-
-### Problem
-
-After installing the Flutter agent and waiting at least 5 minutes, **No HTTP data appears:** on the **HTTP errors** and **HTTP requests** UI pages. 
-
-### Solution
+## Install the Flutter agent [#install]
 
-If no HTTP data appears after you wait at least five minutes, check you are not overriding `HttpOverrides.global` inside your Flutter app.
+To install the Flutter agent:
 
-</Callout>
+* Follow our [guided install](https://onenr.io/0gR70k5oZjo), located directly in the UI.
 
```

Comment:
Same comment here about removing the bullet point and combining it into just one sentence.